src/nrc/api/viewsets.py

```diff
@@ -84,8 +88,14 @@ class DomainViewSet(
     create:
     Defines a new domain with its basis properties and filter attributes.
 
```

Comment:
Hier (of onder de `update` documentatie) willen we ook nog een stukje voor `partial_update`